templates/statefulset-zabbix-proxy.yaml

```diff
@@ -44,7 +44,11 @@ spec:
 {{- toYaml .Values.zabbixagent.resources | nindent 12 }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
+          {{- if .Values.zabbix_image_tag }}
+          image: "{{ .Values.zabbixagent.image.repository }}:{{ .Values.zabbix_image_tag }}"
+          {{- else }}
           image: "{{ .Values.zabbixagent.image.repository }}:{{ .Values.zabbixagent.image.tag }}"
```

Comment:
Hi @sa-ChristianAnton!

With tag centralized, this line and the if can be removed, correct?

Reply:
commit 3f529c7